integration/auction_test.sh

```diff
@@ -19,24 +20,11 @@ auction_test() {
     expect_switcheroo_fail=$1
 
     # install, init, and register (auction) chaincode
-    # install some dummy chaincode (we manually need to create the image)
-    try ${PEER_CMD} chaincode install -n ${CC_ID} -v ${CC_VERS} -p github.com/hyperledger/fabric/examples/chaincode/go/example02/cmd
+    try ${PEER_CMD} chaincode install -l fpc-c -n ${CC_ID} -v ${CC_VERS} -p github.com/hyperledger-labs/fabric-private-chaincode/ecc/
```

Comment:
why not calling it `fpc-cpp`? I think at the moment we can also just call it `fpc` as we do not support any other chaincode language as c/cpp. This may of course change later (far future (next gen release)) with other chaincode languages support. So now I would call it just `fpc`

Reply:
cpp sounds to me like c pre-processor but i guess you mean c++?  I realize that we don't currently support any other language but i still think it's useful to mention the language, after all the param is called --lang.  What about fpc-c++  if we want to emphasize the ++ aspect of c?

Reply:
Once we have the SDKization in place, will we the enduser install the chaincode using `-p github.com/hyperledger-labs/fabric-private-chaincode/examples/auction`?

Reply:
that was more or less the thought with -p. Everything should be in place in this PR and is more or less designed to work with the changes from Bruno's PR in mind